app/assets/javascripts/import_model_library.js.erb

```diff
@@ -0,0 +1,56 @@
+var target_url = "<%= URLResolver.getUrl("import_model_library_interactives_path") %>"
+var lightboxconfig={
+        content:"<div style='padding:10px'>Loading...Please Wait.</div>",
+        title:"Import Model Library",
+        width:400,
+        height:160
+    },import_model_library_modal;
+
+function ajax_success(data){
+   import_model_library_modal.handle.setContent(data);
+};
```

Comment:
I would wrap this whole file in self-executing function, so we don't expose too many unnecessary globals. `ajax_success` can be risky, especially if this code is based on some existing one. :wink: Also, small thing, but I would keep CamelCase naming, typical whitespace (` ` before `{` etc.) and fix semicolons.